### PR TITLE
msx2_flop.xml: Added 56 items, 52 working. Removed 1 item.

### PR DESCRIPTION
**Pull request by wilbertpol**
Renamed “NV Magazine Special 26 (Japan)” to “NV Magazine 1996-7 (Japan)”.
Renamed “NV Magazine 1996-08 (Japan)” to “NV Magazine 1996-8 (Japan)”.
Renamed “NV Magazine 1996-09 (Japan)” to “NV Magazine 1996-9 (Japan)”.
Renamed “NV Magazine 1997-01 (Japan)” to “NV Magazine 1997-1 (Japan)”.
Renamed “NV Magazine 1997-09 (Japan)” to “NV Magazine 1997-9 (Japan)”.
Renamed “NV Magazine 2002-06 (Japan)” to “NV Magazine 2002-6 (Japan)”.
Renamed “NV Magazine 2002-07 (Japan, incomplete dump)” to “NV Magazine 2002-7 (Japan)”.
Renamed “NV Magazine 2002-08 (Japan)” to “NV Magazine 2002-8 (Japan)”.
Removed “R・SYSTEM 3.2 (Japan)”, it was part of NV Magazine 1996-12.

New working software list items
-------------------------------
NV Hokkaido Vol. 1 (Japan) [file-hunter]
NV Magazine 1 (Japan) [file-hunter]
NV Magazine 2 (Japan) [file-hunter]
NV Magazine #3 (Japan) [file-hunter]
NV Magazine 4 (Japan) [file-hunter]
NV Magazine #5 (Japan) [file-hunter]
NV Magazine #7 (Japan) [file-hunter]
NV Magazine #8 (Japan) [file-hunter]
NV Magazine #9 (Japan) [file-hunter]
NV Magazine #10 (Japan) [file-hunter]
NV Magazine #20 (Japan) [file-hunter]
NV Magazine #21 (Japan) [file-hunter]
NV Magazine #22 (Japan) [file-hunter]
NV Magazine #23 (Japan) [file-hunter]
NV Magazine #24 (Japan) [file-hunter]
NV Magazine #26 (Japan) [file-hunter]
NV Magazine 1996-10 (Japan) [file-hunter]
NV Magazine 1996-11 (Japan) [file-hunter]
NV Magazine 1998-6 (Japan) [file-hunter]
NV Magazine 1998-9 (Japan) [file-hunter]
NV Magazine 1998-10 (Japan) [file-hunter]
NV Magazine 1999-1 (Japan) [file-hunter]
NV Magazine 1999-2 (Japan) [file-hunter]
NV Magazine 1999-3 (Japan) [file-hunter]
NV Magazine 1999-5 (Japan) [file-hunter]
NV Magazine 1999-6 (Japan) [file-hunter]
NV Magazine 1999-7 (Japan) [file-hunter]
NV Magazine 1999-8 (Japan) [file-hunter]
NV Magazine 1999-9 (Japan) [file-hunter]
NV Magazine 1999-10 (Japan) [file-hunter]
NV Magazine 1999-11 (Japan) [file-hunter]
NV Magazine 2000-1 (Japan) [file-hunter]
NV Magazine 2000-2 (Japan) [file-hunter]
NV Magazine 2000-3 (Japan) [file-hunter]
NV Magazine 2000-4 (Japan) [file-hunter]
NV Magazine 2000-5 (Japan) [file-hunter]
NV Magazine 2000-6 (Japan) [file-hunter]
NV Magazine 2000-7 (Japan) [file-hunter]
NV Magazine 2000-8 (Japan) [file-hunter]
NV Magazine 2000-9 (Japan) [file-hunter]
NV Magazine 2000-10 (Japan) [file-hunter]
NV Magazine 2000-11 (Japan) [file-hunter]
NV Magazine 2000-12 (Japan) [file-hunter]
NV Magazine 2002-1 (Japan) [file-hunter]
NV Magazine 2002-2 (Japan) [file-hunter]
NV Magazine 2002-4 (Japan) [file-hunter]
NV Magazine 2002-5 (Japan) [file-hunter]
NV Magazine SP #2 (Japan) [file-hunter]
NV Magazine Special #3 (Japan) [file-hunter]
NV Magazine Special #4 (Japan) [file-hunter]
NV Magazine Special #6 (Japan) [file-hunter]
NV RSDM #4 (Japan) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
NV Magazine 1996-12 (Japan) [file-hunter]
NV Magazine 1998-11 (Japan) [file-hunter]
NV Magazine 2001-12 (Japan) [file-hunter]
NV Magazine 2002-3 (Japan) [file-hunter]

### CODE REVIEW

File: hash/msx2_flop.xml [hunk 1]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン2号"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 2 - syntax.dsk" size="737280" crc="656259a0" sha1="2b707101e0afd063f17eb67ae6e5d54772a81890"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx2_flop.xml:22174]: Please use “1-gou” and “2-gou” in the descriptions (or “1-gō” and “2-gō” if this software list uses that style).

File: hash/msx2_flop.xml [hunk 2]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+		<year>1992</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン#3"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 4 - syntax.dsk" size="737280" crc="ccf812c5" sha1="ee5f3d652e33a7d15d6f16134076e0bb9b8a9c93"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx2_flop.xml:22200]: Description and Japanese title don’t match – Japanese title is a duplicate of `nvmg3`.

File: hash/msx2_flop.xml [hunk 3]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+				<rom name="nv magazine 1996-12 - syntax (disk e).dsk" size="737280" crc="d714a9b1" sha1="53e792ba8462879bb1a45bf60350e2252800f18a"/>
+			</dataarea>
+		</part>
+		<!-- ディスクカタログ - disk catalogue -->
+		<part name="flop6" interface="floppy_3_5">
+			<feature name="part_id" value="Disk F"/>
+			<dataarea name="flop" size="737280">
+				<rom name="nv magazine 1996-12 - syntax (disk f).dsk" size="737280" status="nodump"/>
+			</dataarea>
+		</part>
```
Comment (cuavas) [inline hash/msx2_flop.xml:22670]: If this is on the disk label, it should go in the `part_id` so it’s visible to users.

File: hash/msx2_flop.xml [hunk 4]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+	</software>
+
+	<software name="nvmgsp3">
+		<description>NV Magazine Special #3 (Japan)</description>
+		<year>1994</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン増刊号#3"/>
+		<info name="usage" value="Requires a Japanese system."/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:23634]: Please use “NV Magazine Zoukan-gou #3” (or “NV Magazine Zōkan-gō #3”).

File: hash/msx2_flop.xml [hunk 5]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmgsp4">
+		<description>NV Magazine Special #4 (Japan)</description>
+		<year>1995</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン増刊号#4"/>
+		<info name="usage" value="Requires a Japanese system."/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:23660]: Please use “NV Magazine Zoukan-gou #4 (or “NV Magazine Zōkan-gō #4”).

File: hash/msx2_flop.xml [hunk 6]
```diff
@@ -22145,196 +22134,1591 @@ HOMEBREW
...
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nvmgsp6">
+		<description>NV Magazine Special #6 (Japan)</description>
+		<year>1996</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="NVマガジン増刊号#6"/>
+		<info name="usage" value="Requires a Japanese system."/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:23686]: Please use “NV Magazine Zoukan-gou #6” (or “NV Magazine #6”).

**Review by cuavas — COMMENTED** [review]
Besides the inconsistent titles I noticed, have you checked the accuracy of the titles for these disks in general?

File: hash/msx2_flop.xml (hunk 3 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:22670]: There are no images of the disks themselves, so I do not know if that text was on the disk label.

In the selection menu it is referenced by the name 'ディスクカタログ' and indicated to be on Disk F. The comment is purely informational to help with finding/identifying this disk and should be removed if the disk ever gets found.

nvmg9612

**Comment by wilbertpol** [comment]
> Besides the inconsistent titles I noticed, have you checked the accuracy of the titles for these disks in general?

I executed each of these disks to see the titles used. And through time they changed the format of the titles and sometimes even the names used by different disks of the same release were different.

File: hash/msx2_flop.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml]: Fixed

File: hash/msx2_flop.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml]: Fixed

File: hash/msx2_flop.xml (hunk 4 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml]: Fixed

File: hash/msx2_flop.xml (hunk 5 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml]: Fixed

File: hash/msx2_flop.xml (hunk 6 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml]: Fixed

**Pull request merged**